README.md

```diff
@@ -93,6 +123,8 @@ What it sets up
 [Tmux]: http://tmux.sourceforge.net/
 [Zsh]: http://www.zsh.org/
 [Oh My ZSH!]: http://ohmyz.sh/
+[ZSH Syntax Highlighting]: https://github.com/zsh-users/zsh-syntax-highlighting
+[Z]: https://github.com/rupa/z
```

Comment:
`z` is pretty sweet... but `j` is not part of the laptop script anyway. I don't think there's any reason they can't both be added, but one doesn't preclude the other.